### PR TITLE
feat: setup detail page

### PR DESCRIPTION
**Pull request by theogainey**
### Description
Add detail page as [id].jsx
Use URL slug, `next/router`, and `useRouter` hook to determine what info to show
No new tests were added because new functionality that should not break with future pull requests comes directly from `next/router` & `useRouter` and not new code contributed in this pull request. 

### Spec

See Story: [FSA22V1-162]((https://sparkbox.atlassian.net/browse/FSA22V1-162))

### Validation

* [ ] This PR has code changes, and our linters still pass.

#### To Validate

1. Make sure all PR Checks have passed (GitHub Actions, CircleCI, Code Climate, Snyk, etc).
2. Pull down all related branches.
3. Confirm all tests pass.
4. Start dev server `npm run dev`
5. Navigate to  `http://localhost:3000/<someName>`
6. Visually confirm that `Hello someName!` appears

### CODE REVIEW

File: pages/[id].jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { useRouter } from 'next/router';
```
Comment (dustin-jw) [inline pages/[id].jsx:1]: I asked Meg to include a <Head> component in her PR, so I think we should be consistent and have one here as well, so we'll want to `import Head from 'next/head';` here.

File: pages/[id].jsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+export default function DetailPage() {
+  const router = useRouter();
+  const { id } = router.query;
+
+  return (
+    <h1>
+      {`Hello ${id}!`}
+    </h1>
```
Comment (dustin-jw) [inline pages/[id].jsx:10]: For the sake of making it easier to add things later and being consistent with the other PR, let's add in a little more html structure.

```suggestion
    <div>
      <Head>
        <title>{`${id} | Sparkbox Team Availability`}</title>
        <meta name="description" content={`View details about ${id}, including their projects, skills, and interests.`} />
      </Head>

      <main>
        <h1>
          {`Hello ${id}!`}
        </h1>
      </main>
    </div>
```

Comment (dustin-jw) [inline pages/[id].jsx:10]: For the sake of making it easier to add things later and being consistent with the other PR, let's add in a little more html structure.

```suggestion
    <div>
      <Head>
        <title>{`${id} | Sparkbox Team Availability`}</title>
        <meta name="description" content={`View details about ${id}, including their projects, skills, and interests.`} />
      </Head>

      <main>
        <h1>
          {`Hello ${id}!`}
        </h1>
      </main>
    </div>
```

**Review by dustin-jw — CHANGES_REQUESTED** [review]
Looks good! I just have a few requests to make this a notch above "Hello world".

File: pages/[id].jsx (hunk 1 above)
Comment (theogainey) [reply pages/[id].jsx:1]: Good suggestion. I added it in

File: pages/[id].jsx (hunk 2 above)
Comment (theogainey) [reply pages/[id].jsx]: I agree, these changes make sense.

**Review by dustin-jw — APPROVED** [review]
Nice work! I nominate @arnest00 to finalize the review!

**Review by arnest00 — APPROVED** [review]
Looks good to me!

**Pull request merged**